tests/main/boot-state/task.yaml

```diff
@@ -43,42 +43,42 @@ execute: |
             ;; 
     esac
 
-    # check a new var can be set in bootenv
+    echo "Check a new variable can be set with  bootenv set command"
     "$TESTSTOOLS"/boot-state bootenv show | NOMATCH 'snap_test_1='
     "$TESTSTOOLS"/boot-state bootenv set snap_test_1 test_1
     "$TESTSTOOLS"/boot-state bootenv show | MATCH 'snap_test_1=test_1$'
 
-    # check a var can be showed in bootenv
+    echo "Check a new variable is correctly displayed with bootenv show command"
     "$TESTSTOOLS"/boot-state bootenv set snap_test_2 test_2
     "$TESTSTOOLS"/boot-state bootenv show snap_test_1 | MATCH 'test_1'
     "$TESTSTOOLS"/boot-state bootenv show snap_test_1 | NOMATCH 'test_2'
 
-    # check a var can be set even if it is already defined in bootenv
+    echo "Check a variable can be set even if it is already defined in bootenv"
     "$TESTSTOOLS"/boot-state bootenv set snap_test_1 test_3
     "$TESTSTOOLS"/boot-state bootenv show snap_test_1 | MATCH 'test_3'
     test "$("$TESTSTOOLS"/boot-state bootenv show | grep -c snap_test_1)" -eq 1
 
-    # check an existing var can be unset in bootenv
+    echo "Check an existing variable can be unset with bootenv unset command"
     "$TESTSTOOLS"/boot-state bootenv unset snap_test_1
     "$TESTSTOOLS"/boot-state bootenv show | NOMATCH 'snap_test_1='
     "$TESTSTOOLS"/boot-state bootenv show | MATCH 'snap_test_2=test_2$'
     "$TESTSTOOLS"/boot-state bootenv unset snap_test_2
     "$TESTSTOOLS"/boot-state bootenv show | NOMATCH 'snap_test_2='
 
-    # check an inexistent var can be unset in bootenv
+    echo "check an inexistent var can be unset in bootenv"
     "$TESTSTOOLS"/boot-state bootenv unset snap_boot_no_exist
 
-    # wait-core-post-boot should finish inmediatly
+    echo "Check that the wait-core-post-boot command finishes inmediatly"
     "$TESTSTOOLS"/boot-state wait-core-post-boot
     # TODO: Test the scenario when the core reaches the timeout running wait-core-post-boot
 
-    # check bootenv needs a subcommand
+    echo "Check bootenv shows an error message if not subcommand is used"
     "$TESTSTOOLS"/boot-state bootenv 2>&1 | MATCH "boot-state: unsupported bootenv sub-command"
     "$TESTSTOOLS"/boot-state bootenv noexist 2>&1 | MATCH "boot-state: unsupported bootenv sub-command noexist"
 
-    # check var and value are needed to call bootenv set
+    echo "Check the bootenv set command shows an error message if variable and value are not used"
     "$TESTSTOOLS"/boot-state bootenv set 2>&1 | MATCH "boot-state: variable and value required to set in bootenv"
     "$TESTSTOOLS"/boot-state bootenv set justvar 2>&1 | MATCH "boot-state: variable and value required to set in bootenv"
 
-    # check var is needed to call bootenv unset
+    echo "Check the bootenv unset command shows an error message if a variable is not used"
```

Comment:
s/variable is not used/variable name is missing/?

Reply:
done